src/renderers/__tests__/ReactCompositeComponent-test.js

```diff
@@ -504,8 +504,8 @@ describe('ReactCompositeComponent', () => {
 
     expectDev(console.error.calls.count()).toBe(1);
     expectDev(console.error.calls.argsFor(0)[0]).toBe(
-      'Warning: defaultProps was defined as an instance property ' +
-        'on Component. Use a static property to define defaultProps instead.',
+      'Warning: setting defaultProps as an instance property on Component is not supported ' +
```

Comment:
Thanks for clarifying! Your comment above had it as lowercase so I thought it was fine, but I'll make note of this from now on.